pyproject.toml

```diff
@@ -68,6 +64,9 @@ archs = ["auto", "aarch64"]
 [tool.cibuildwheel.macos]
 archs = ["x86_64", "universal2", "arm64"]
 
+[tool.cibuildwheel.macos.environment]
+MACOSX_DEPLOYMENT_TARGET = "10.15"
```

Comment:
Is it because `std::filesystem::path` is only available after 10.15? Can we document this?

Reply:
I moved this change to #74 
We need this env variable to enable >=C++17